Toolset/palettes/menubar/revmenubar.livecodescript

```diff
@@ -710,13 +710,21 @@ on updateButtonHilite pTargetName, pValue
    lock screen
    local tTargetLongID
    put the long id of button pTargetName of group "toolbar" of me into tTargetLongID
+   
+   local tShowIcon
+   put the showIcon of tTargetLongID into tShowIcon
+   
    setButtonHilite tTargetLongID, pValue
    set the label of tTargetLongID to revMenubarButtonText(pTargetName, pValue)
    if pValue then
       set the icon of tTargetLongID to the cHighlightedIcon of tTargetLongID
    else
       set the icon of tTargetLongID to the cIcon of tTargetLongID
    end if
+   
+   if not tShowIcon then
+      set the showIcon of tTargetLongID to false
+   end if
```

Comment:
Could you unilaterally `set the showIcon of tTargetLongID to tShowIcon` here, without the test?